.eslintrc.json

```diff
@@ -31,13 +45,42 @@
   },
   "overrides": [
     {
-      "files": ["**/__tests__/*.{j,t}s?(x)", "**/tests/unit/**/*.spec.{j,t}s?(x)"],
+      "files": [
+        "**/tests/*.{j,t}s?(x)",
+        "**/tests/unit/**/*.{j,t}s?(x)"
+      ],
       "env": {
         "jest": true
       },
+      "globals": {
+        "badRequest": true,
+        "cardService": true,
+        "io": true,
+        "ready1": true,
+        "request": true,
+        "socket1": true,
+        "socket2": true,
+        "socket3": true
+      },
```

Comment:
Due to the way you instantiate your tests with the `bootstrap.test.js` file I can not figure out a better way to make these globals not throw a `no-undef` error so I just opted to add them here as _known_ globals for these files.

Reply:
Okay interesting. The unit tests with supertest aren't actually even used in CI so it may be best to simply throw them out altogether and rely entirely on e2e with cypress. I'll do some digging